src/rust/engine/Cargo.toml

```diff
@@ -147,3 +151,5 @@ env_logger = "0.5.4"
 prost = { git = "https://github.com/danburkert/prost", rev = "a1cccbcee343e2c444e1cd2738c7fba2599fc391" }
 prost-build = { git = "https://github.com/danburkert/prost",  rev = "a1cccbcee343e2c444e1cd2738c7fba2599fc391" }
 prost-types = { git = "https://github.com/danburkert/prost",  rev = "a1cccbcee343e2c444e1cd2738c7fba2599fc391" }
+# TODO: Stabilize before landing.
+console = { git = "https://github.com/stuhood/console", branch = "stuhood/term-target-for-arbitrary-handles" }
```

Comment:
Any reason not to pull this code into the pants project yet?

Reply:
Yea, I'll fork them to `pantsbuild/pants` before landing.